101-aci-dynamicsnav/azuredeploy.json

```diff
@@ -99,6 +105,14 @@
                 { 
                   "name": "PublicDnsName",
                   "value": "[variables('publicdnsname')]"
```

Comment:
this needs to come from the reference() function - as this is written will fail when the location parameter is used

Reply:
@bmoore-msft I think we've had that discussion here https://github.com/Azure/azure-quickstart-templates/pull/4616/files/a3380d5851b8caad0ce09faafe22412ed178b5c7 before or am I misunderstanding you? I need to have the fqdn when the container starts and don't know how to achieve that

Reply:
Alright - we need to find a solution for this... we're not going to have this pattern repeated in the repo.  I would recommend finding out where that re-entrant requirement is coming from and see how that can be addressed.

Reply:
@bmoore-msft The only thing I can come up with would be the following:

1. Run a script at the very beginning of container startup which calls an Azure Function
2. That Azure Function tries to take the requesting IP (and I hope this is the public IP of the container)
3. I run something like nslookup to determine the FQDN of that IP

Kind of crude but I don't see an alternative. Do you see any obvious problems with that approach